docs/source/installation.rst

```diff
@@ -207,6 +207,22 @@ Some users have experienced issues when installing NumPy, particularly version 1
     Another option under Windows is to follow the instructions for `Unix-like OS`_, using the Windows Subsystem for Linux (WSL).
     For installing WSL, follow the instructions `here <https://docs.microsoft.com/en-us/windows/wsl/install-win10#step-2---check-requirements-for-running-wsl-2>`_.
 
+Some quick tips on virtual environments
+---------------------------------
+
+As mentioned above -  we recommend that you do all developement for sktime within a virtual environment.  Two good options for doing so are
```

Comment:
avoid duplications, text should be crisp. Header should indicate what this section is about.

Reply:
Makes sense, I moved this section before the `Setting up a development environment` and removed references to that.

Reply:
I think there's better ways to address the comments - the new section is very verbose. Good instruction manuals outline clear short steps.

Think of it as writing code. You are executing code "on the reader", i.e., telling them what to do. It is not very clear what to do from the new section. There is an "if using vscode, elif using sth else", e.g., which is not too clear.

The second section also seems to be a condition for the first section, because you need to have your python environment before using it in notebooks! So it seems, at least in parts, the wrong way round...